packages/flutter_tools/lib/src/commands/ios.dart

```diff
@@ -0,0 +1,164 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
```

Comment:
nit: global nit - flutter style arbitrarily recommends using `'` instead of `"` for strings.